frame/balances/src/lib.rs

```diff
@@ -1051,6 +1059,18 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 		});
 		Ok(actual)
 	}
+
+	/// Check invariants are consistent within the pallet.
+	/// - `total_issuance` should be less than `Config::Balance::max_value()`.
+	#[cfg(any(test, feature = "try-runtime", feature = "fuzz"))]
+	pub fn do_try_state() -> Result<(), &'static str> {
```

Comment:
We can also make a check that the sum of all balances is equal to total issuance.. but that will take AGES. 

@ggwpez what do you think? red or blue pill? :D

Reply:
So red pill? :) @ggwpez.

Reply:
It is outside of the scope of this MR. Just an idea to add a resolution/granularity level to the `try_state` hook which defines how many checks should run. Or rather how long the checks should take in total.  
The wen could only run the slower tests for the higher granularity setting.

Reply:
Ohh I see. 

So the red pill as @kianenigma suggested? @ggwpez.

Reply:
I think you can check it. The try-state hooks take very long currently already and we can probably parallelize them.

Reply:
If I understand what you mean correctly, you are saying let there be tiers in the operation that goes on in try_state hooks, which determines the number of tests that are allowed to be tested and if it happens to be slower (`higher granularity`). Place it somewhere else to be executed? @ggwpez